static/js/click_handlers.js

```diff
@@ -791,6 +792,7 @@ exports.initialize = function () {
 
         $("body").on("click", "[data-finish-editing]", function (e) {
             const selector = $(this).attr("data-finish-editing");
+            $(selector).css({"text-overflow": "ellipsis", "border-style": ""});
```

Comment:
We don't generally add manual CSS like this in JavaScript code; all styling should be done in CSS.  Can you change this logic to add/remove a CSS class (if there isn't one already) that controls this behavior.

It'd also be helpful if you wrote in your commit message the reasoning behind why you think this is the right fix.

Reply:
Added the add/remove class functionality. 

Regarding the border addition- it provides a textbox-like area to edit the name.
Regarding the ellipsis to clip during editing- ellipsis prevented editing the name (added the screenshot in the new commit message).